packages/web-react/src/types/fileUploader.ts

```diff
@@ -20,13 +20,28 @@ export interface FileUploaderTextProps {
   linkText?: string;
 }
 
+export interface FileQueueValueMetaType {
+  [key: string | number]: unknown;
+}
+
+export interface FileQueueValueType {
+  file: File;
+  meta?: FileQueueValueMetaType;
+}
+
+export interface UpdateQueueBaseType {
```

Comment:
This type does not extend the type above. They both have partially the same values. There should be and `extend` used, I think.